### PR TITLE
Update schedule with upcoming releases

### PR DESCRIPTION
**Pull request by dsimansk**
# Changes

- Update schedule with upcoming releases

@pierDipi @creydr if you already have an idea for the next Eventing release leads, feel free to add the names. I can alter between Serving and Eventing as well. Or not be there at all. :)

/cc @knative/knative-release-leads

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ The Knative project releases quarterly, on Tuesday of the 4th week of January, A
 
 | Release | Release Date | Serving             | Eventing                        | PKG cut    | Unpin repos
 | ------- | ------------ | ------------------- | --------------------------------| ---------- | -----------
-| v1.9    | 2023-01-24   | psschwei            | evankanderson                   | 2023-01-17 | 2023-01-25
-| v1.10   | 2023-04-25   | pradnyavmw, kvmware | dsimansk, pierDipi              | 2023-04-18 | 2023-04-26
 | v1.11   | 2023-07-25   | skonto, dsimansk    | creydr, pierDipi, Vishal-Chdhry | 2023-07-18 | 2023-07-26
-| v1.12   | 2023-10-24   | TBD                 | TBD                             | 2023-10-17 | 2023-10-25
+| v1.12   | 2023-10-24   | skonto, dsimansk    | TBD                             | 2023-10-17 | 2023-10-25
```
Comment (dsimansk) [inline README.md:21]: @skonto per you comment, I kept it the same for the next one for now.

**Comment by ReToCode** [comment]
@dsimansk you can also add me for Serving. I'll shadow the next one to get into this for Serving.

**Comment by dsimansk** [comment]
> @dsimansk you can also add me for Serving. I'll shadow the next one to get into this for Serving.

Cool, thanks! Done.

**Review by ReToCode — APPROVED** [review]
/lgtm

**Pull request merged**